catalog/resourcemanagement/resourcemanagement-usage-ui/src/main/webapp/js/view/DataUsage.view.js

```diff
@@ -84,7 +84,9 @@ define([
                this.listenTo(this.model, 'change:monitorLocalSources', this.render);
             },
             onRender : function() {
-               this.setupPopOver('[data-toggle="update-all-popover"]', 'Updates the Data Limit for all users in the table.  If there are individual fields set in the table, the limit for all fields takes precedence.');
+               this.setupPopOver('[data-toggle="update-all-popover"]',
+                 "Updates the Data Limit for all users in the table. [-1] indicates unlimited data usage. [0] indicates data usage is prohibited. " +
+                 "If there are individual fields set in the table, the limit set for all users takes precedence.");
```

Comment:
That makes sense. How about "If this value is changed, it will be set as the data limit for all users even if limits for specific users have also been set"?